gui/mountPrimaryWindow.py

```diff
@@ -135,21 +144,13 @@ def showUnderConstructionPage(self):
 
 
     def mountBucket(self):
-        self.addOutputText("Validating configuration...")
-        # Update the pipeline/components before mounting the target
-        targetIndex = self.dropDown_bucketSelect.currentIndex() 
-        success = self.modifyPipeline(bucketOptions[targetIndex])
-        if not success:
```

Comment:
This is the code I was thinking about, which gates mount with modifyPipeline.
I see that the new way you're doing it (attaching the function to the dropdown), doesn't really allow for this gate here. So I think a backlog item for that to be added, either to the GUI or the CLI, is fine.